gradle.properties

```diff
@@ -1,5 +1,5 @@
 # ---versions--- #
-version=3.0-SNAPSHOT
+version=3.1-SNAPSHOT
```

Comment:
Didn't we roll back to 3.0?

Reply:
Yes it was accidently pushed by rbasing onto main